vhdbuilder/packer/test/run-test.sh

```diff
@@ -63,14 +63,26 @@ else
 
     IMG_DEF="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
   else 
-    #gen2Mode check, set the IMG_DEF to the MANAGED_SIG_ID retrieved from packer-output after VHD Build
-    IMG_DEF=${MANAGED_SIG_ID}
+    # gen2Mode check, set the IMG_DEF to the MANAGED_SIG_ID retrieved from packer-output after VHD Build
+    if [ "$OS_TYPE" == "Linux" ]; then
+      IMG_DEF=${MANAGED_SIG_ID}
+    else
+      # (to-do) for Windows: MANAGED_SIG_ID reads as "file"
```

Comment:
do we still need this comment?

Reply:
No. `IMG_DEF=${MANAGED_SIG_ID}` for both Linux and Windows.